src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -217,6 +217,7 @@ internal PackageSourceMoniker SelectedSource
 
         public INuGetExperimentationService NuGetExperimentationService { get; private set; }
 
+        public bool ShowVulnerabilities => _topPanel.CheckBoxVulnerabilities.IsChecked == true;
```

Comment:
Is anything in `_topPanel.CheckBoxVulnerabilities.IsChecked` nullable? I'm guessing you don't need `== true`.

Reply:
I'm guessing you meant to remove the `== true` check?

Reply:
_topPanel.CheckBoxVulnerabilities can be null 😢 I need to leave this